MyPiggyBank.Data/Model/OperationCategory.cs

```diff
@@ -6,7 +6,10 @@ namespace MyPiggyBank.Data.Model {
     public class OperationCategory
     {
         public Guid Id { get; set; }
-        public string UserEmail { get; set; }
+        public Guid UserId { get; set; }
```

Comment:
Trzymajmy się jednej konwencji. Jeśli w innych klasach na przykład "Operation". 
`       public virtual Resource Resource { get; set; }
        public virtual OperationCategory OperationCategory { get; set; }`
Nie dodawaliśmy `public Guid ResourceId {get;set;}` to tutaj też nie dodajemy czegoś takiego. 
Chyba, że dodajemy w klasie "Operation"
`public Guid ResourceId {get;set;}
 public Guid OperationCategory {get;set;}`.
Tak samo w innych klasach. Ja proponuje cos takiego dodać do tamtych klas, bo na przykład "ktoś" robiąc linq to sql będzie mógł w łatwy sposób łączyć tabele. Inna sytuacja jest jeśli używamy linq to sql fluent.